o2a/o2a_libs/el_wf_functions.py

```diff
@@ -60,8 +74,14 @@ def last_error_node(context=None, session=None) -> str:
     if not last_failed_task:
         return ""
 
-    name: str = last_failed_task.task_id
-    return name
+    task_name: str = last_failed_task.task_id
+
+    task_map: Optional[dict] = context.get("task_map", None)
+    if task_map is None:
+        raise AirflowException("No task map!")
+
+    reversed_map: dict = _reverse_task_map(task_map)
+    return reversed_map.get(task_name, "")
```

Comment:
This allows us to return original Oozie node name, thus preserving any possible logic.